orca/service_impl.go

```diff
@@ -0,0 +1,173 @@
+/*
```

Comment:
`service.go` and drop the `_impl`?  It's cleaner.

Reply:
Done.